detection/luna16_prepare_env_files.py

```diff
@@ -45,20 +59,37 @@ def main():
     env_dict = {}
     env_dict["orig_data_base_dir"] = raw_data_base_dir
     env_dict["data_base_dir"] = resampled_data_base_dir
-    env_dict["data_list_file_path"] = os.path.join(downloaded_datasplit_dir,"original/dataset_fold0.json")
+    env_dict["data_list_file_path"] = os.path.join(
+        downloaded_datasplit_dir, "original/dataset_fold0.json"
+    )
+    if dicom_meta_data_csv != None:
+        env_dict["dicom_meta_data_csv"] = dicom_meta_data_csv
     with open(out_file, "w") as outfile:
         json.dump(env_dict, outfile, indent=4)
 
-
     # generate env json file for training and inference
     for fold in range(10):
-        out_file = "config/environment_luna16_fold"+str(fold)+".json"
+        out_file = "config/environment_luna16_fold" + str(fold) + ".json"
         env_dict = {}
-        env_dict["model_path"] = os.path.join(out_trained_models_dir,"model_luna16_fold"+str(fold)+".pt")
+        env_dict["model_path"] = os.path.join(
+            out_trained_models_dir, "model_luna16_fold" + str(fold) + ".pt"
+        )
         env_dict["data_base_dir"] = resampled_data_base_dir
+<<<<<<< HEAD
```

Comment:
some git operations should be removed...please also check the other files
```suggestion

```

Reply:
Sorry! I just cleared them